tools/api_boost/api_boost.py

```diff
@@ -132,7 +132,6 @@ def ApiBoostTree(target_paths,
     sp.run([
         'bazel',
         'build',
-        '--config=libc++',
```

Comment:
Sorry just so I understand the comments above, it seems like we used to use libc++? Why can't we use it now?

Reply:
This is part of `bazel.api` RBE conversion. The flags are now propagated through BAZEL_BUILD_OPTIONS, because `--config=libc++` doesn't work in RBE, it has to be `--config=rbe-toolchain-clang-libc++`.

For API booster it doesn't really has to be on libc++ based build, it's easier to just let it follow what BAZEL_BUILD_OPTIONS says.

Reply:
OK got it, thanks for the explanation.